component-generator/templates/test.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import ComponentName from './index';
+import componentName from './index';
```

Comment:
reverting the changes done in one of the other PRs, this actually caused component generator to generate incorrect tests, it's important that `componentName` is in camelCase